packages/core/src/components/ozon-content/ozon-content.tsx

```diff
@@ -1,48 +1,199 @@
-import { h, Event, Component, ComponentInterface, Prop, Listen, EventEmitter } from '@stencil/core';
-
-import { ContentAnchor } from './ozon-content.interfaces';
-import { OzonContentTransformer } from './ozon-content.transformer';
+import {
+  Component,
+  ComponentInterface,
+  Event,
+  EventEmitter,
+  Fragment,
+  h,
+  Prop,
+} from "@stencil/core";
+import { ContentAnchor } from "./ozon-content.interfaces";
+import { v4 as uuid } from "uuid";
 
 @Component({
-  tag: 'dso-ozon-content',
-  styleUrl: 'ozon-content.scss'
+  tag: "dso-ozon-content",
+  styleUrl: "ozon-content.scss",
+  shadow: true,
 })
 export class OzonContent implements ComponentInterface {
   @Prop()
-  content!: string;
+  content: string | undefined;
 
   @Event()
   anchorClick!: EventEmitter<ContentAnchor>;
 
-  @Listen('click')
-  handleClick(event: PointerEvent) {
-    this.transformer.handleClickEvent(event);
-  }
+  getAttribute = (node: Node, name: string): string => {
+    return node instanceof Element ? node.getAttribute(name) ?? "" : "";
```

Comment:
Is dit waterdicht? Wat nou als het echt type `Node` is.. Of is alles stiekem `instanceof Element` ?

Reply:
Het doet zijn ding daar waar ik het nodig heb. Zolang dat zo blijft is het water- en luchtdicht.

Reply:
Oki 🤷‍♂️